code/src/components/SignIn.spec.tsx

```diff
@@ -9,19 +9,33 @@ jest.mock('next/navigation', () => ({
 }));
 
 describe('SignInForm', () => {
-  test('renders SignInForm', () => {
-    const { getByPlaceholderText } = render(
-      <SignIn
-        email={undefined}
-        setEmail={() => {}}
-        password="password"
-        setPassword={() => {}}
-      >
-        children
-      </SignIn>,
+  test('should render login form when status is true', () => {
+    const { getByRole, getByPlaceholderText } = render(<SignInForm status />);
```

Comment:
If you want to test SignInForm you should do it in SignInForm spec, it's a different component